service/hook/github/github.go

```diff
@@ -120,10 +121,11 @@ func transformPushEvent(pushEvent PushEventModel) hookCommon.TransformResultMode
 			TriggerAPIParams: []bitriseapi.TriggerAPIParamsModel{
 				{
 					BuildParams: bitriseapi.BuildParamsModel{
-						Branch:          branch,
-						CommitHash:      headCommit.CommitHash,
-						CommitMessage:   headCommit.CommitMessage,
-						PushCommitPaths: pushEvent.CommitPaths,
+						Branch:            branch,
+						CommitHash:        headCommit.CommitHash,
+						CommitMessage:     headCommit.CommitMessage,
+						PushCommitPaths:   pushEvent.CommitPaths,
+						BaseRepositoryURL: pushEvent.Repo.getRepositoryURL(),
```

Comment:
This might return the SSH URL, it's fine by me, but I think @mateherber mentioned it will always return the http one, not sure what you're expecting in the related step.

Reply:
We handle both. If it returns the ssh, so be it :)